test/verify/check-system-terminal

```diff
@@ -350,6 +350,68 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
         self.wait_line(2, "/var")
         self.assertIn("No such process", m.execute(f"kill -0 {pid} 2>&1 || true"))
 
+    def testExternalState(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/playground/terminal")
+
+        b.wait_text('.pf-v6-c-toggle-group__button.pf-m-selected', "Terminal 1")
+
+        # Run something in Terminal 1
+        b.wait_in_text(line_sel(1), "admin@local")
+        b.input_text("echo Terminal 1\n")
+        b.wait_text(line_sel(2), "Terminal 1")
+
+        # Switch to Terminal 2
+        b.click('.pf-v6-c-toggle-group__button:contains(Terminal 2)')
+        b.wait_text('.pf-v6-c-toggle-group__button.pf-m-selected', "Terminal 2")
+        b.wait_text_not(line_sel(2), "Terminal 1")
+
+        # Run something else there
+        b.wait_in_text(line_sel(1), "admin@local")
+        b.input_text("echo Terminal 2\n")
+        b.wait_text_not(line_sel(2), "Terminal 2")
```

Comment:
That `_not` looks wrong. It only passed the test as the wait_not is quick enough to not see the last key strokes yet.